core/src/util.ts

```diff
@@ -4,3 +4,15 @@ export const LOGGER = getLogger("@ailly/core");
 LOGGER.level = getLogLevel(process.env["AILLY_LOG_LEVEL"]);
 
 export const isDefined = <T>(t: T | undefined): t is T => t !== undefined;
+
+Promise.withResolvers =
+  Promise.withResolvers ??
```

Comment:
Looks like something Jiffies would have.

Reply:
2021, yes. This landed in Node 18; jiffies targets 20+. This is only in Ailly for AL2 platforms on Node16.